hplsql/src/test/results/local/create_package3.out.txt

```diff
@@ -1,5 +1,5 @@
 Ln:1 INCLUDE src/test/queries/local/create_package3_include.sql
 INCLUDE CONTENT src/test/queries/local/create_package3_include.sql (non-empty)
-EXEC PACKAGE PROCEDURE A.test
-Ln:9 PRINT
-"test ok"
\ No newline at end of file
+EXEC PACKAGE PROCEDURE A.TEST
+Ln:8 PRINT
```

Comment:
line number was correctly 9 earlier. I wonder why that changed to 8.

Reply:
It's because the package header + body stored in distinct fields the they're put back together when loading. This can eat some trailing newlines from the end.